mantidproject/mantid
# Indirect Bayes - Sort X axis

The x-axis of the output workspaces is now sorted.

**To test:**

[SortX.zip](https://github.com/mantidproject/mantid/files/1004478/SortX.zip)

Indirect>Bayes>Quasi
Load files provided
press run
Check that in `sample_QLd_Prob` and `sample_QLd_Result` the x-axis is sorted.

repeat with Bayes>Stretch


Fixes #18969 .



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
